lib/runtime/printk.c

```diff
@@ -12,10 +12,12 @@
  * init time. If no routine is installed, a nop routine is called.
  */
 
+#include <kernel.h>
 #include <misc/printk.h>
 #include <stdarg.h>
 #include <toolchain.h>
 #include <linker/sections.h>
+#include <kernel_runtime.h>
 
 typedef int (*out_func_t)(int c, void *ctx);
```

Comment:
The implementations for this function pointer return void instead of int.

Reply:
derp, fixed